File: Day 34/homework/hw5.py
# 5) გაიხსენეთ გაკვეთილზე ახსნილი ახალი ოპერატორი (%). დაწერეთ პროგრამა, სადაც მომხმარებელმა უნდა შემოიტანოს ინტეჯერი რიცხვი. if-else ების გამოყენებით დაადგინეთ შემოტანილი რიცხვი ლუწი იქნება თუ კენტი. თუ რიცხვი ლუწია დაბეჭდეთ 'The number is even'. ხოლო თუ კენტია დაბეჭდეთ 'The number is odd'.


number = int(input("Enter your number: "))

if number % 2 and  number != 0:
    print("The number is odd")
else:
    print("The number is even")

File: Day 138/homework/hw5.js
// 5) შექმენით Form, სადაც დაამატებთ age input-ს. დაამატეთ ვალიდაცია, რომ შეამოწმოთ მომხმარებელს მხოლოდ რიცხვები შემოჰყავს თუ არა. თუ ეს კრიტერიუმი არ შესრულდება - alert-ით გამოიტანეთ შესაბამისი მესიჯი. მომხმარებელს არ მისცეთ ფორმის დადასტურების საშუალება თუ ამ კრიტერიუმს არ შეასრულებს.


const form = document.getElementById("form");

form.addEventListener("submit", (e) =>{
    e.preventDefault();

    const age = document.getElementById('age').value;

    if(age.length === 0){
        alert("You must enter your age!!")
        return;
    }

    if(isNaN(age)){
        alert('age must be a Number!!')
        return;
    }

})